packages/gatsby/src/utils/api-runner-node.js

```diff
@@ -76,6 +77,7 @@ const runAPI = (plugin, api, args) => {
         reporter,
         getNodeAndSavePathDependency,
         cache,
+        createNodeId,
```

Comment:
I can't remember if we talked about this but we should automatically set the namespace for each plugin using its name. That way there's no chance that ids could clash.

Reply:
Oh, that was mentioned before, yeah.  Obviously not here yet, but I'll add it quick.

Reply:
Ha, actually it was half here, the method takes a namespace, just wasn't setting that up properly.